.mci.yml

```diff
@@ -346,12 +333,10 @@ functions:
                   export UNIFIED_FORMAT_TESTS_PATH=$(pwd)/../data/unified-format
 
                   # export environment variables for encryption tests (and redact them)
```

Comment:
You should probably remove "(and redact them)" since the redaction was necessary due to all the xtrace business, but that has been removed.